docs/content/integrations/dbt/using-dbt-with-dagster/downstream-assets.mdx

```diff
@@ -58,7 +58,10 @@ To add the `order_count_chart` asset:
 2. After your definition of `jaffle_shop_dbt_assets`, add the definition for the `order_count_chart` asset:
```

Comment:
worth mentioning why we're adding compute_kind here too, since this is the first time in the guide it's introduced.